fml/raster_thread_merger.cc

```diff
@@ -18,6 +18,10 @@ RasterThreadMerger::RasterThreadMerger(fml::TaskQueueId platform_queue_id,
       task_queues_(fml::MessageLoopTaskQueues::GetInstance()),
       lease_term_(kLeaseNotSet) {
   is_merged_ = task_queues_->Owns(platform_queue_id_, gpu_queue_id_);
+  if (is_merged_) {
+    // lease_term_ must be > 0 if the threads are merged.
```

Comment:
would you mind expanding on this? Why does it need to be > 0?

Reply:
done